### ISSUE TITLE
Accessibility support - unable to tab (i.e. swipe) to that button

### ISSUE THREAD

**Issue by tahsin-mostafiz** [mrchandoo/cordova-plugin-decimal-keyboard#18]
Hello, 

I love the idea behind your plugin, and it worked perfectly until I was about to implement the accessibility requirements. 

I have tried a few of the different accessibility attributes, but it seems that it just won't get added to the list of accessibility elements. 

Any suggestions? 

Aside, I modified your native code to adjust for safeAreaLayoutGuide in iPhone X. I'll make a PR when I get a chance. Thanks in advance!

**Comment by mrchandoo** [comment]
@tahsin-mostafiz are you sending a PR ?
